app/scenes/login/login.js

```diff
@@ -39,11 +58,58 @@ class Login extends Component {
         }
     }
 
-    signIn() {
+    preSignIn = () => {
+        this.setState({error: null});
```

Comment:
Question, what's this call to setState doing? Doesn't look like ```this.state.error``` is being read anywhere from within ```preSignIn```.